metadata-ingestion/src/datahub/ingestion/source/looker/lookml_concept_context.py

```diff
@@ -382,14 +402,45 @@ def _get_list_dict(self, attribute_name: str) -> List[Dict]:
             return ans
         return []
 
+    def _include_parent_fields(
```

Comment:
is this a static method? which fields is this fetching, or is it fetching all fields?

Reply:
I renamed the method to better show its responsibility `merge_parent_and_child_fields` and moved it out of class